### PR TITLE
Infrastructure: further unify webpack config

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

While working on #6918, I noticed a couple of places where this config could be improved. I have listed in the improvements below. 

## Relevant Technical Choices

The big different between bundles now, is what name of the progress bar and if it does and does not use bundled pollyfills / react. 

With this shared config, it now generate RTL version of CSS for every bundle. 

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

File: webpack.config.cjs [hunk 1]
```diff
@@ -325,24 +316,14 @@ const activationNotice = {
   },
   plugins: [
     ...sharedConfig.plugins,
+    new DependencyExtractionWebpackPlugin({
```
Comment (spacedmonkey) [inline webpack.config.cjs:322]: Should we be using the built in version of react ( wp-elements ) for activation?

**Comment by swissspidy** [comment]
> This PR addresses an existing issue and I have linked this PR to it in ZenHub

I don't see an issue this is linked to, even though the checkbox is ticked.

**Comment by spacedmonkey** [comment]
> I don't see an issue this is linked to, even though the checkbox is ticked.

There is no issue, this is just random tweaks.

**Comment by swissspidy** [comment]
> > I don't see an issue this is linked to, even though the checkbox is ticked.
> 
> There is no issue, this is just random tweaks.

Then don't tick it if it's not accurate :-)

**Review by swissspidy — APPROVED** [review]

**Pull request merged**